source/CardGames.py

```diff
@@ -284,6 +285,12 @@ def index(self, player_wanted, info):
       if str(player_wanted) == str(player):
         return i
 
+  def score(self): #scorekeeping function
+    print(self.player_turn.hand)
+    if self.player_turn.MatchFour(): #if player has a set of 4
```

Comment:
great use of the MatchFour function, nice job being able to keep their wins available for them to see and to just add onto their existing wins. one thing you could have done was to create a screen that displays everyones scores but other than that it looks fine 👍